test/e2e/shared/defaults.go

```diff
@@ -51,6 +51,7 @@ const (
 	FlavorV1alpha5              = "v1alpha5-ci-artifacts"
 	FlavorMDRemediation         = "md-remediation-ci-artifacts"
 	FlavorKCPRemediation        = "kcp-remediation-ci-artifacts"
+	FlavorFlatcar               = "flatcar-ci-artifacts"
```

Comment:
It is a bit strange to add the "ci-artifacts" here since we skip injecting the script for it.